docs/docs/features/studio.md

```diff
@@ -0,0 +1,109 @@
+---
+title: ZMK Studio
+sidebar_label: Studio
+---
+
+:::warning[Alpha Feature]
+
+ZMK Studio support is in alpha. Although best efforts are being made, keeping compatibility during active development is not guaranteed.
+
+:::
+
+ZMK Studio provides runtime update functionality to ZMK powered devices. Studio is still under active development, and is not yet ready for casual end user use.
```

Comment:
It might be nice to "dumb down" `runtime update functionality` a bit. Also, though not necessary, this could be a nice place to put an image or short video of ZMK Studio in use.

Reply:
Expanded on this just a bit.